tests/jestified/mvn-parsed-tree-to-depgraph.spec.ts

```diff
@@ -0,0 +1,191 @@
+import { parseTrees } from '../../lib/parse/tree';
+import { createFromJSON } from '@snyk/dep-graph';
+import { parseStdout } from '../../lib/parse/stdout';
+import {
+  smallVerboseDepGraphData,
+  largeVerboseDepGraphData,
+} from './fixtures/verbose-depgraph';
+import { verboseAggregatedProjects } from './fixtures/verbose-aggregated-projects';
+
+it('small - output from mvn dependency:tree -Dverbose to be parsed from tree to depgraph', () => {
```

Comment:
this isn't testing reality is it? we use `-DoutputType=dot` which has a very different output

Reply:
(wheres the inspect test that would have uncovered this)